docs/rules/prefer-stateless-function.md

```diff
@@ -6,7 +6,7 @@ Stateless functional components are simpler than class based components and will
 
 This rule will check your class based React components for
 
-* methods/properties other than `displayName`, `propTypes`, `render` and useless constructor (same detection as ESLint [no-useless-constructor rule](http://eslint.org/docs/rules/no-useless-constructor))
+* methods/properties other than `displayName`, `propTypes`, `contextTypes`, `defaultProps`, `render` and useless constructor (same detection as ESLint [no-useless-constructor rule](http://eslint.org/docs/rules/no-useless-constructor))
```

Comment:
awesome, i was going to ask about contextTypes :-)

Reply:
Doesn't seem to be any tests for contextTypes.

Reply:
If you felt like adding them here, that'd be super awesome :-D

Reply:
Oops. I thought I saw them, but it was `childContextTypes`. I added tests for `contextTypes` to this.